oracledb-ballerina/client.bal

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+public client class Client {
+    *sql:Client;
+    private boolean clientActive = true;
+
+    # Initialize Oracle Client.
+    #
+    # + host - Hostname of the oracle server to be connected
+    # + user - Name of a user of the database
+    # + password - Password for the user
+    # + database - System Identifier or the Service Name of the database
+    # + port - Port number of the oracle server to be connected
+    # + options - Oracle database specific JDBC options
```

Comment:
We should not say `JDBC` here, as the purpose of DB specific modules is not to expose JDBC stuff, but to have a generic connector even which we can use  with native impl of ballerina in future.

Reply:
Fixed with 28468e1a9bbdfaf47b9d0a048879f548137638ca